test/e2e/cypress.json

```diff
@@ -1,5 +1,6 @@
 {
 	"baseUrl": "http://localhost:8088",
-	"ignoreTestFiles": "**/*.json",
-	"video": false
+	"testFiles":	"**/*.js",
+	"video": false,
+	"env": {"stack" : "test_ab"}
```

Comment:
Have we updated the installer to manage the stack references here?

Reply:
Yes, I have a draft PR  digi-serve/ab-cli#20. I'll test it works after I merge this in.